src/app/layout/header/header.component.less

```diff
@@ -96,6 +96,30 @@
   border-right-color: @color-pf-black-600;
 }
 
+#header_createSpace {
+  display: block;
+  .create-space-anchor {
+    background-color: #fff;
+    transition: all .3s;
+    border-color: transparent;
+    border-style: solid;
+    border-width: 1px 0;
+    padding: 1px 10px;
+    display: block;
+    clear: both;
+    font-weight: 400;
+    line-height: 1.667;
+    color: @color-pf-black-800;
+    white-space: nowrap;
+    &:hover {
+      border-color: @color-pf-blue-100;
+      background-color: @color-pf-blue-50;
+      text-decoration: none;
+      color: @color-pf-black-700;
+    }
+  }
+}
+
```

Comment:
Not clear why this is needed. Is it because the `f8-feature-toggle` tag was added between the `li` and `a` tags? If so, can the `li` tag be wrapped instead?

We also don't want to attach CSS styles to IDs. Can we use create-space-anchor instead?

Reply:
@dlabrecq Yes Dan. That's the only reason for including extra CSS.
Wrapping might not help because the styles from patternfly and other generic areas appear like below

```
.navbar-pf .dropdown-menu>li>a
```

and a few similar styles of this sort.

As it is trying to find the (immediate child) via selector, this anchor won't have those properties known.  

To your point about not having to use Id, yes it makes sense and it has now been changed to class.